adapters/gamma/gamma.go

```diff
@@ -180,6 +201,28 @@ func (a *GammaAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapte
 	return adapterRequests, errs
 }
 
+func convertBid(gBid gammaBid, mediaType openrtb_ext.BidType) *openrtb.Bid {
+	var bid openrtb.Bid
+	bid = gBid.Bid
+
+	if mediaType == openrtb_ext.BidTypeVideo {
+		//Return inline VAST XML Document (Section 6.4.2)
```

Comment:
This branch (where mediaType == openrtb_ext.BidTypeVideo) has no test coverage. Can you please add tests to cover this new logic?